bindings/wasm/lib/util.test.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2025 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// import {strict as assert} from 'assert';
+import {expect, suite, test} from 'vitest';
+
+import {transformStaticImportsToDynamic} from './util.ts';
+
+suite('transformStaticImportsToDynamic', () => {
+  test('import foo from "bar"', () => {
+    const input = 'import foo from "bar";';
+    const result = transformStaticImportsToDynamic(input);
+    expect(result).to.equal('const {default:foo} = await import(\'bar\');');
+  });
+
+  test('import { foo } from "bar"', () => {
+    const input = 'import { foo } from "bar";';
+    const result = transformStaticImportsToDynamic(input);
+    expect(result).to.equal('const { foo } = await import(\'bar\');');
+  });
+
+  test('import { foo, bar } from "baz"', () => {
+    const input = 'import { foo, bar } from "baz";';
+    const result = transformStaticImportsToDynamic(input);
+    expect(result).to.equal('const { foo, bar } = await import(\'baz\');');
+  });
+
+  test('import { foo /*, bar */ } from "baz"', () => {
+    const input = 'import { foo /*, bar */ } from "baz";';
+    const result = transformStaticImportsToDynamic(input);
+    expect(result).to.equal('const { foo } = await import(\'baz\');');
```

Comment:
What happened to bar?

Reply:
`bar` is commented out.  This test is to make sure that comments are not evaluated.

The user never sees the transformed code, so stripping the comment doesn't affect readability.  At some point I imagine I'll have to add source map generation to this module, but that's a later issue.